demo/runtime/src/lib.rs

```diff
@@ -295,6 +296,44 @@ impl frame_system::Config for Runtime {
 	type PostTransactions = ();
 }
 
+#[dynamic_params(RuntimeParameters, pallet_parameters::Parameters::<Runtime>)]
+pub mod dynamic_params {
+	use super::*;
+
+	#[dynamic_pallet_params]
+	#[codec(index = 0)]
+	pub mod cardano_config {
+		#[codec(index = 0)]
+		pub static SecurityParameter: u64 = 432;
+		#[codec(index = 1)]
+		pub static ActiveSlotsCoeff: sp_runtime::Permill = sp_runtime::Permill::from_percent(5);
+	}
+
+	#[dynamic_pallet_params]
+	#[codec(index = 1)]
+	pub mod cardano_epoch_config {
+		#[codec(index = 0)]
+		pub static EpochDuration: core::time::Duration =
+			core::time::Duration::from_millis(432000000); // 5 days
+		#[codec(index = 1)]
+		pub static SlotDurationMillis: core::time::Duration =
+			core::time::Duration::from_millis(1000); // 1 second
```

Comment:
https://github.com/IntersectMBO/cardano-node/blob/master/configuration/cardano/mainnet-shelley-genesis.json 4 params are in this file